### PR TITLE
Rules changed that look'ed at target value to not be unknown.

### PR DESCRIPTION
**Pull request by lasrinnil**
… inspected

## Description

We have a few rules which allowed false positive by supplying them with a int value other than the allowed or the unknown option. This PR will address this issue.

## References

* #193

### CODE REVIEW

**Comment by BjarkeMeier** [comment]
Should we add unit testing of the impacted `ValidationRules/OperationTypeValidationRule.cs` with a current coverage of 0%?

**Comment by Mech0z** [comment]
Maybe expand unittest with value outside the enum values

**Comment by lasrinnil** [comment]
> 
> 
> Maybe expand unittest with value outside the enum values

Done!

**Comment by lasrinnil** [comment]
> OperationTypeValidationRule

Added the [UnitTest] Att. to the class :)

**Review by bemwen — APPROVED** [review]

File: source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/ChargeTypeIsKnownValidationRuleTest.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ public class ChargeTypeIsKnownValidationRuleTest
         [InlineAutoMoqData(ChargeType.Fee, true)]
         [InlineAutoMoqData(ChargeType.Subscription, true)]
         [InlineAutoMoqData(ChargeType.Subscription, true)]
```
Comment (prtandrup) [inline source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/ChargeTypeIsKnownValidationRuleTest.cs:31]: Subscription was mentioned twice
```suggestion
        [InlineAutoMoqData(ChargeType.Tariff, true)]
```

Comment (lasrinnil) [reply source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Application/Validation/InputValidation/ValidationRules/ChargeTypeIsKnownValidationRuleTest.cs:31]: done!

**Review by Mech0z — APPROVED** [review]

**Pull request merged**